src/mg/noobframework/utils/MethodUtils.java

```diff
@@ -76,24 +78,38 @@ else if (parameter.getType().equals(Mysession.class)) {
 
     }
 
-    public static Object executMethod(Mapping mapping, HttpServletRequest request, String verb) throws Exception {
+    public static Object executMethod(Mapping mapping, HttpServletRequest request, String verb,
+            HashMap<String, String> error) throws Exception {
         Class<?> clazz = mapping.getClazzMapping();
         Object obj = clazz.getConstructor().newInstance();
         setSessionAttribut(obj, request);
         Method method = mapping.getMethodMapping(verb);
-        List<Object> paramValue = getParamValues(mapping, request, verb);
+        List<Object> paramValue = getParamValues(mapping, request, verb, error);
         return method.invoke(obj, paramValue.toArray());
     }
 
     public static void doMethod(HttpServletRequest request, HttpServletResponse response, Mapping mapping,
             PrintWriter pWriter, String verb) throws Exception {
-        Object result = executMethod(mapping, request, verb);
+        HashMap<String, String> error = new HashMap<>();
+        Object result = executMethod(mapping, request, verb, error);
+
         if (result instanceof String) {
-            // pWriter.println("<p>" + result + "</p>");
             pWriter.println(new Gson().toJson(result));
         } else if (result instanceof Modelview) {
             Modelview modelview = (Modelview) result;
             HashMap<String, Object> data = modelview.getData();
+
+            // CHECK VALIDATION
+            if (error.size() > 0) {
```

Comment:
Refactor this code in so that your codebase is modular. MethodUtils should not contains any redirect that the controller or ErrorHandler should do xD